include/cufinufft/impl.h

```diff
@@ -354,6 +354,115 @@ Notes: the type T means either single or double, matching the
     return ier;
 }
 
+template<typename T>
+T calculate_scale_factor(int rank, finufft_spread_opts opts) 
+{
+    // Calculate the additional scalar for multiplying just for spread and interpolate
+    int n = 100;
```

Comment:
What is n here? This routine looks like a crude quadrature approximation to the integral of the spread kernel. But it is not accurate. A Gauss-Legendre rule would be needed.